packages/block-editor/src/components/inserter/style.scss

```diff
@@ -705,12 +705,9 @@ $block-inserter-tabs-height: 44px;
 	display: none;
 }
 
-.is-zoom-out {
```

Comment:
This style was relying on the existence of some global CSS class, this is not great, especially since this class is define outside the package entirely while we do have access to the mode within the package.